NEWS.md

```diff
@@ -8,6 +8,12 @@
 
 ### New Features
 
+* Function name changes:
+  * `read_redcap_tidy` is now `read_redcap`
```

Comment:
Please always include `()` at the end of function names in docs, e.g. `read_redcap()` not `read_redcap`.

Reply:
Done!